.travis.yml

```diff
@@ -5,6 +5,8 @@ before_install:
 script:
         - ./make.sh
         - make check
+        - cp libcapstone.so bindings/python/
```

Comment:
this "libcapstone.so" limits our test to Linux only. OSX has "libcapstone.dylib", can we support that?